cmd/releasego/update-azure-linux.go

```diff
@@ -274,13 +290,50 @@ func generateUpdateBranchNameFromAssets(assets *buildassets.BuildAssets) string
 	return fmt.Sprintf("refs/heads/update-go-%s", assets.GoVersion().Full())
 }
 
-func generatePRTitleFromAssets(assets *buildassets.BuildAssets) string {
-	return fmt.Sprintf("Bump Go Version to %s", assets.GoVersion().Full())
+func generatePRTitleFromAssets(assets *buildassets.BuildAssets, security bool) string {
+	var b strings.Builder
+	if security {
+		b.WriteString("(security) ")
+	}
+	b.WriteString("golang: bump Go version to ")
+	b.WriteString(assets.GoVersion().Full())
+	return b.String()
 }
 
-func GeneratePRDescription(assets *buildassets.BuildAssets) string {
-	const format = "Bump Go Version to %s"
-	return fmt.Sprintf(format, assets.GoVersion().Full())
+func GeneratePRDescription(assets *buildassets.BuildAssets, security bool, notify string, prNumber int) string {
+	// Use a builder for readability. Use fmt.Fprintf in situations when it would be hard to read
```

Comment:
nit: I've never heard about this convention. IMO it looks cleaner to always use the same API, i.e. `fmt.Fprintf` for formating and 'fmt.Fprint` for writing raw strings.